### ISSUE TITLE
support BLS12-381

### ISSUE THREAD

**Issue by xhliu** [sCrypt-Inc/snarkjs#4]
Export sCrypt verifier for curve bls12-381, besides BN256.

For more info, refer to https://github.com/iden3/snarkjs. Start with https://github.com/iden3/snarkjs#1-start-a-new-powers-of-tau-ceremony
`snarkjs powersoftau new bn128 12 pot12_0000.ptau -v`
->
`snarkjs powersoftau new bls12-381 12 pot12_0000.ptau -v`

**Comment by xhliu** [comment]
See https://github.com/sCrypt-Inc/snarkjs/commit/7df0083c083406fae2d5f999566f773240c4fa8a on how to support BN256 and https://github.com/sCrypt-Inc/snarkjs/commit/a8b1d6733994e17c9ade642ca4d4a32af8c0b5bd for PLONK.

**Comment by msinkec** [comment]
The verifier is now added in 780fde11d49da9bbb6f5dfdd5afd90dad52a5104. I tried supporting it with miller pre-calculation, however in an WASM context we are limited to Zokrates ark backend to achieve that (See https://github.com/Zokrates/ZoKrates/issues/1200). Values from the ark backend don't work for now (https://github.com/sCrypt-Inc/zokrates/issues/35) so the generated sCrypt code doesn't use the optimization for now. 

I created a separate issue for the optimization: https://github.com/sCrypt-Inc/snarkjs/issues/7 
I'm closing this one.
